scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -281,7 +281,8 @@ object ScalafmtConfig {
   )
 
   val dotty: ScalafmtConfig = default.copy(
-    runner = ScalafmtRunner.default.copy(dialect = scala.meta.dialects.Dotty)
+    runner =
+      ScalafmtRunner.default.copy(dialect = ScalafmtRunner.Dialect.Scala3)
```

Comment:
I found another issue with indentation :/ I think we can always rebase all other PR depending on which gets merged first.

Reply:
i merged your other pr, time to rebase?

Reply:
Rebased!